torchtext/csrc/clip_tokenizer.cpp

```diff
@@ -1,5 +1,5 @@
-#include <clip_tokenizer.h>
-#include <regex.h> // @manual
+#include "clip_tokenizer.h"
+#include "regex.h" // @manual
```

Comment:
I bet these changes have consequences in fbcode. So you might want to check if this change works in fbcode at one point before merging this PR.

Reply:
Yup will import this WIP PR to fbcode first before I merge into OSS

Reply:
n00b question. Why do we need to make this change?

Reply:
tldr: otherwise the compiler cannot find the headers.

The previous build process was adding the `torchtext/csrc` to list of system header directories, which is not quite right but was working fine and torchtext C++ headers are not meant to be used from C++ application so it was not a problem. Now CMake is not doing that, so the visibility of header files have changed.